src/client/thirdParty/Inkeep.tsx

```diff
@@ -0,0 +1,97 @@
+'use client';
+
+import { InkeepChatButtonProps } from '@inkeep/uikit';
+import type {
+  InkeepAIChatSettings,
+  InkeepSearchSettings,
+  InkeepBaseSettings,
+  InkeepModalSettings,
+} from '@inkeep/uikit';
+import { env } from '../../env';
+import dynamic from 'next/dynamic';
+
+/**
+ * Inkeep (AI Help) chat button
+ * Implemented according to https://docs.inkeep.com/integrations/nextjs/chat-button
+ */
+
+const ChatButton = dynamic(() => import('@inkeep/uikit').then((mod) => mod.InkeepChatButton), {
+  ssr: false,
+});
+
+type InkeepSharedSettings = {
+  baseSettings: InkeepBaseSettings;
+  aiChatSettings: InkeepAIChatSettings;
+  searchSettings: InkeepSearchSettings;
+  modalSettings: InkeepModalSettings;
+};
+
+const useInkeepSettings = (): InkeepSharedSettings => {
+  const apiKey = env.NEXT_PUBLIC_INKEEP_API_KEY;
+  const integrationId = env.NEXT_PUBLIC_INKEEP_INTEGRATION_ID;
+  const organizationId = env.NEXT_PUBLIC_INKEEP_ORG_ID;
+
+  const baseSettings: InkeepBaseSettings = {
+    apiKey,
+    integrationId,
+    organizationId,
+    primaryBrandColor: '#F04405',
+    //logEventCallback: customAnalyticsCallback,
+  };
+  const modalSettings: InkeepModalSettings = {};
+  const searchSettings: InkeepSearchSettings = {};
+  const aiChatSettings: InkeepAIChatSettings = {
+    chatSubjectName: 'Fingerprint',
+    botAvatarSrcUrl: 'https://fingerprint.com/img/uploads/fpjs-small-logo-for-email.png',
```

Comment:
Is it a permanent icon?

Reply:
Hey @ilfa, yes it is. This is the logo we want to keep using when opening inkeep. Afaik there's no intention to change it.

Reply:
Ah, I get the question now. That's a fair question which I do not know the answer to. I was told to use this icon and seems to live in the public application mostly for email. I'm not sure for how long it has been there, but I need a good enough icon that is hosted online. Maybe as an improvement I can try publishing the same image on this repo, but I can do this separately as an improvement, appreciate you mentioning that!